OJP/OJP_FareSupport.xsd

```diff
@@ -556,6 +556,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareProductPriceGroup"/>
+			<xs:element name="AvailableTicketsAtPrice" type="xs:positiveInteger" minOccurs="0">
```

Comment:
This means that 0 is not available. With zero you don't show this ticket, as in you won't tell (ever) that it is unavailable?